### PR TITLE
Bump generator versions and add support for --name-mappings

### PR DESCRIPTION
**Pull request by DmitrySboychakov**

### CODE REVIEW

**Comment by tjarvstrand** [comment]
@DmitrySboychakov A new version of the OpenAPI generator and cli were just released. Please consider updating this PR so we can get the latest version right away.

It contains some fixes we need so it would be greatly appreciated.

Thanks

**Comment by DmitrySboychakov** [comment]
@tjarvstrand yes, I am aware. I'd like to see the review from @gibahjoe first and I can prepare re-worked and updated PR

**Comment by quyenvsp** [comment]
Thank you but could you please remove binary file? As I only trust binary file from author

**Comment by DmitrySboychakov** [comment]
@quyenvsp not a problem, but I am awaiting for review from @gibahjoe first.

**Comment by gibahjoe** [comment]
Sorry guys, got busy with studies. This PR looks good. Just a review to get through

**Review by gibahjoe — APPROVED** [review]

File: openapi-generator-annotations/lib/src/openapi_generator_annotations_base.dart [hunk 1]
```diff
@@ -521,14 +529,6 @@ class DioProperties extends AdditionalProperties {
...
 
 class DioAltProperties extends AdditionalProperties {
-  /// Changes the minimum version of Dart to 2.12 and generate null safe code
-  final bool? nullSafe;
-
-  /// nullSafe-array-default
-  /// Makes even arrays that are not listed as being required in your OpenAPI "required"
-  /// but making them always generate a default value of []
-  final bool? nullSafeArrayDefault;
-
```
[the remark this answers is not in the document]
Comment (DmitrySboychakov) [reply openapi-generator-annotations/lib/src/openapi_generator_annotations_base.dart:531]: yes, they are removed

**Pull request merged**